avogadro/qtplugins/importpqr/pqrrequest.h

```diff
@@ -107,7 +107,8 @@ private slots:
     result()
       : inchikey("Error"), name("Error"), mol2url("Error"), formula("Error"),
         mass(-1.0)
-    {}
+    {
```

Comment:
These kind of changes seem really weird considering a bunch of other short blocks are consolidated.

If we can get rid of these, I'd appreciate it.